### PR TITLE
Added new names after update

### PR DESCRIPTION
**Pull request by ArxdSilva**
screen shot 2016-03-13 at 6 33 47 pm

### CODE REVIEW

File: names.json [hunk 1]
```diff
@@ -40,6 +43,7 @@
   "Ganso",
   "Jeba",
   "Jenga",
+  "Jibóa",
```
Comment (joaocunha) [inline names.json]: `Jibóia`?

**Comment by ArxdSilva** [comment]
for some reason when I try "node test" it will try to find "test" file and wont run the test, only runs if you go with "node test.js"

**Comment by joaocunha** [comment]
@ArxdSilva that might be a config issue. Will have a look.

Thanks a lot! We reached :100: :eggplant:!

I'm getting :cocktail: tonight :P

**Comment by ArxdSilva** [comment]
https://instagram.com/p/BC6RrzQzJtB/

**Pull request merged**